.buildkite/pipeline.yml

```diff
@@ -17,3 +17,14 @@ steps:
       - NIX_PATH=nixpkgs=$(nix eval --raw '(import ./nix/sources.nix).nixpkgs') nix-shell -p curl git gitAndTools.hub --run "curl https://raw.githubusercontent.com/serokell/scratch/release-binary/scripts/release-binary.sh | bash"
     label: Create a pre-release
     branches: master
+  - command:
+      - nix-build docker
+      - NIX_PATH=nixpkgs=$(nix eval --raw '(import ./nix/sources.nix).nixpkgs') nix run nixpkgs.skopeo -c skopeo copy --dest-creds "serokell:${DOCKERHUB_PASSWORD}" "docker-archive:$(readlink result)" "docker://docker.io/serokell/xrefcheck:latest"
+    label: Push to dockerhub
+    branches: master
+  - command:
+      - nix-build docker
+      - NIX_PATH=nixpkgs=$(nix eval --raw '(import ./nix/sources.nix).nixpkgs') nix run nixpkgs.skopeo -c skopeo copy --dest-creds "serokell:${DOCKERHUB_PASSWORD}" "docker-archive:$(readlink result)" "docker://docker.io/serokell/xrefcheck:${BUILDKITE_BRANCH}"
```

Comment:
1. Where does this `${DOCKERHUB_PASSWORD}` come from?
2. Do I understand correctly that everything is supposed to happen automatically and we shouldn't bother uploading anything to Docker ourselves?

Reply:
1. From vault. If you need more info, ask in Slack
2. Yes. If you want to, you can upload manually, but there should be no reason to -- it uploads `latest` from master and `v<...>` from respective tags.